### DISCUSSION TITLE
Modrinth Page?

### DISCUSSION THREAD

**Discussion by PolyframeX** [P3pp3rF1y/SophisticatedBackpacks#865] [General]
It would be very helpful if a modrinth page was available. This would make the mod much more accessible to users, and make it more widespread.

**Comment by gingershaped** [comment]
I'm making a modpack on Modrinth and would really like to include Sophisticated Backpacks, any news on this being a thing?

**Comment by RaeTheGit** [comment]
I'd like to second this. CurseForge has become quite hostile to anything outside of the Windows/CF/Twitch diaspora and is by far the most versatile and well designed backpack mod that I know of. It would be a delight seeing this on Modrinth too.

For now I suppose the best method is to inculde it as an override in packs that are not published on CF.

**Comment by MaxTheMooshroom** [comment]
I 3rd this, as a pack developer.

**Comment by Zangdorx** [comment]
I fourth this, as a pack dev too

**Comment by ThatKnownStranger** [comment]
I would also really appreciate this mod being available on Modrinth!

**Comment by B1BU** [comment]
P3pp3rF1y's mods are basically the only ones I use in my server that aren't avaliable on Modrith yet, I really wish they would start posting there

**Reply by MaxTheMooshroom** [reply]
Same here... As well as Chisels & Bits, they're the only mods I use that aren't on Modrinth

**Comment by DDunda** [comment]
I would also love to see this mod on Modrinth, it is such an essential so it's a shame it is only on CurseForge. I'm sure packs would quickly move include it

**Comment by PinkDisorder** [comment]
So this has been open for nearly a year with no developer response. Is there a reason in particular to gatekeep this mod off modrinth?

**Reply by B1BU** [reply]
Could be just the dev procrastinating on it. I personally plan to post my datapacks on Modrinth but I've been procrastinating on it for months because it's so boring to set up a whole new page with new assets and stuff on a different platform for each one of your projects

**Comment by epicacrylic** [comment]
All of the Sophisticated mods are on Modrinth now

https://modrinth.com/mod/sophisticated-core
https://modrinth.com/mod/sophisticated-backpacks
https://modrinth.com/mod/sophisticated-storage

Officially? I haven't found out yet, but they're there nonetheless

**Reply by P3pp3rF1y** [reply]
Those are official uploads automated from sophisticated repositories so yup this closes the thread here

**Comment by P3pp3rF1y** [comment]
As mentioned in the last reply Sophisticated mods are now officially on Modrinth, the upload is automated so any new release automatically goes on Modrinth in addition to CurseForge. Closing the discussions on this.

https://modrinth.com/mod/sophisticated-core
https://modrinth.com/mod/sophisticated-backpacks
https://modrinth.com/mod/sophisticated-storage

**Reply by PinkDisorder** [reply]
Thank you for putting them on modringh!
